engines/sci/graphics/font.cpp

```diff
@@ -38,7 +38,9 @@ GfxFontFromResource::GfxFontFromResource(ResourceManager *resMan, GfxScreen *scr
 
 	_resource = resMan->findResource(ResourceId(kResourceTypeFont, resourceId), true);
 	if (!_resource) {
-		error("font resource %d not found", resourceId);
+		// The Lighthouse demo does not contain any fonts
```

Comment:
@lskovlun The embedded system font has been there since at least SCI2, and is what I expected to see in the demo, but I don’t think it is code worth adding to support a non-interactive demo. Anyone who wants to run this demo can just open 2.VMD in VLC and get the same experience.